src/spatialexperiment/io/tenx_visium.py

```diff
@@ -0,0 +1,307 @@
+"""Creates a ``SpatialExperiment`` from the Space Ranger output directories for 10x Genomics Visium spatial gene expression data"""
+
+from typing import List, Union, Optional
+from warnings import warn
+import os
+import re
+import json
+import pandas as pd
```

Comment:
move pandas to the specific functions that uses it. SPE itself doesn't need pandas to work, its only the visium importer